### ISSUE TITLE
feat: Add integration with Xero (accounting)

### ISSUE THREAD

**Issue by rflihxyz** [panoratech/Panora#452]
### Contact Details

rf@panora.dev

### Category

Accounting

### Platforms

Platform : Xero
Objects:
Read vs Write:

**Comment by Dozie2001** [comment]
Hey @rflihxyz  is this task still open? would love to be assigned to it. I am also looking to join the core team as an engineer(won't mind a junior position) what do I have to make that happen

**Comment by rflihxyz** [comment]
Hey @Dozie2001 , are you still interested in working on this issue? We plan to hire in the next 6 months and contributing is the best way to show us your skills, much more than any resume!

**Comment by Dozie2001** [comment]
Yes I am but I just got assigned the first let me work on that first.

**Comment by rflihxyz** [comment]
We haven't deployed our standard data model for this, I would recommend you to wait a few days, 

looping in @naelob

**Comment by myj009** [comment]
@rflihxyz Is this issue open to be worked on now?
I can see zero in the metadata file
